examples/va/nvidia-vfio-passthrough/README.md

```diff
@@ -0,0 +1,90 @@
+`pci_passthrough:alias` flavor extra specs.
+
+### Guest VM
+
+To use the passthrough GPU, the guest operating system inside the VM must have
+the appropriate native NVIDIA driver installed. You will need a standard NVIDIA
+driver. Do not use vGPU-enabled guest drivers. The GPU will appear as a physical
+PCI device within the guest.
+
+### Host Configuration (`examples/va/nvidia-vfio-passthrough/edpm/nodeset/values.yaml`)
+
+The following parameters are crucial for host-level configuration:
+
+*   **BareMetalHost configuration**: `baremetalhosts` section contains information required by metal3 to provision baremetal nodes.
+    *   `bmc.address`: The IP address of the Baseboard Management Controller (BMC).
+    *   `bootMACAddress`: The MAC address of the network interface that the node will use to PXE boot.
+    *   `rootDeviceHints`: Hints for metal3 to identify the root device for the OS installation.
+    *   `preprovisioningNetworkData`: Static nmstate network config to be applied to a `BaremetalHost` via ironic-python-agent ramdisk during provisioning. The config is embedded in the ISO attached as virtual media via the BMC, so no DHCP is required.
+    *   `baremetalHostsNetworkData`: Final nmstate network configuration for EDPM nodes.
+
+*   `edpm_kernel_args`: Appends necessary kernel arguments for VFIO passthrough.
+    *   `intel_iommu=on iommu=pt`: Enables the IOMMU for device passthrough.
+    *   `vfio-pci.ids=10de:20f1`: Instructs the `vfio-pci` driver to claim the specified GPU(s) by their vendor and product IDs at boot time. The example IDs `10de:20f1` are for an NVIDIA A100 GPU.
+    *   `rd.driver.pre=vfio-pci`: Avoids race conditions during boot by loading vfio-pci kernel module early.
+
+*   `edpm_tuned_profile` and `edpm_tuned_isolated_cores`: These parameters configure the `tuned` service.
+    *   `edpm_tuned_profile` is set to `cpu-partitioning-powersave` to enable CPU isolation features.
+    *   `edpm_tuned_isolated_cores` specifies the cores to be isolated. For CPU isolation we strongly recommend using the Tuned approach rather than `isolcpus` kernel argument.
+
+*   **VFIO-PCI Binding Service**: The `vfio-pci-bind` service in `va/nvidia-vfio-passthrough/edpm/nodeset/nova_gpu.yaml` blacklists the `nouveau` and `nvidia` kernel modules to ensure they do not interfere with the `vfio-pci` driver. The service also regenerates the initramfs and grub configuration to apply these changes. A reboot is required for these changes to take effect.
+
+### Nodes
+
+| Role                        | Machine Type | Count |
+| --------------------------- | ------------ | ----- |
+| Compact OpenShift           | vm           | 3     |
+| OpenStack Baremetal Compute | Baremetal    | 1     |
+
+### Networks
+
+| Name         | Type     | Interface | CIDR            |
+| ------------ | -------- | --------- | --------------- |
+| Provisioning | untagged | nic1      | 172.23.0.0/24   |
```

Comment:
I thought there is no `Provisioning` network because you're using virt media instead?